src/Evented.ts

```diff
@@ -36,6 +36,22 @@ function handlesArraytoHandle(handles: Handle[]): Handle {
 	};
 }
 
+export interface EventObject {
+	/**
+	 * The type of the event
+	 */
+	readonly type: string | symbol;
```

Comment:
A good question, I would have tested it but I cannot find any usages of either `EventObject` or `EventedCallback` outside of `Evented` (which suggests that both these interfaces should actually be in the `Evented` package).